README.md

```diff
@@ -129,6 +129,7 @@ Gollum comes with the following command line options:
 | --template-dir    | [PATH]    | Specify custom mustache template directory. |
 | --template-page   | none      | Use _Template in root as a template for new pages. Must be committed. |
 | --emoji           | none      | Parse and interpret emoji tags (e.g. `:heart:`) except when the leading colon is backslashed (e.g. `\:heart:`). |
+| --emoji           | none      | 'Set the default keybinding for editable pages. Can be set to `vim`, or `emacs`. Default: `default`' | 
```

Comment:
Copy/paste issue, should read `--default-keybind`.

Reply:
@fuzzylemma could you make this minor change? Think otherwise this is ready to merge!